### PR TITLE
Set up associations

### PR DESCRIPTION
**Pull request by GutemaG**
#Set up associations

## Relation ships between ```Classroom and Student``` class
- Create ```Classroom class``` with necessary attributes
- Create has-many relationships between ```Classroom and Student class```

## Many to many relationship between ```Person and Book class```
- Create ```Rental and Book Class``` with necessary attributes
- Create Many to many relationships between ```Book and Person``` through ```Rental class```

### CODE REVIEW

File: person.rb [hunk 1]
```diff
@@ -4,14 +4,15 @@
...
-  attr_accessor :name, :age
+  attr_accessor :name, :age, :rental
 
   def initialize(age, name = 'Unknown', parent_permission: true)
     super()
     @id = Random.rand(1..1000)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rental = []
```
Comment (rustam-tolipov) [inline person.rb:16]: - [ ] I can see you initialized the rental in person.rb. However, we need to have a method `add_rental` with (book, date) parameters. Kindly add that!

`# *The person itself should be sent as a parameter to create the new rental achieving the 'has-many' association`

**Review by rustam-tolipov — CHANGES_REQUESTED** [review]
Hello @GutemaG,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
_Your project is almost complete!_
_No Linters errors_
_Perfect associations between methods_

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Check the comments under the review._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: person.rb (hunk 1 above)
Comment (GutemaG) [reply person.rb:16]: @Rustamxon7 thank you
But I need little bit explanation where I have to add the ```add_rental(book, date)``` method. Should I add in both ```person.rb```  and ```book.rb```? why date parameter?

Comment (rustam-tolipov) [reply person.rb:16]: @GutemaG Left message in slack

**Review by CollinsTatang — APPROVED** [review]
# Approved 👏  
### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**